src/Block/FileBlock.php

```diff
@@ -60,17 +60,26 @@ public function getSummary()
     protected function provideBlockSchema()
     {
         $blockSchema = parent::provideBlockSchema();
-        if ($this->File() && $this->File()->exists() && $this->File()->getIsImage()) {
-            $blockSchema['fileURL'] = $this->File()->CMSThumbnail()->getURL();
+        if ($this->File() && $this->File()->exists()) {
             $blockSchema['fileTitle'] = $this->File()->getTitle();
+
+            if ($this->File()->getIsImage()) {
+                $blockSchema['fileURL'] = $this->File()->CMSThumbnail()->getURL();
+            } else {
+                $blockSchema['fileURL'] = ModuleResourceLoader::resourceURL(
+                    'silverstripe/framework:client/images/app_icons/document_92.png'
+                );
+            }
         }
+
         return $blockSchema;
     }
 
     /**
      * Return a thumbnail of the file, if it's an image. Used in GridField preview summaries.
      *
      * @return DBHTMLText
+     * @deprecated 3.1 Will be removed without equivalent functionality to replace it
```

Comment:
I marked method as deprecated here.

Reply:
```suggestion
```
Please remove that - I agree with your assessment that it shouldn't be marked as deprecated until the next minor release which is covered by https://github.com/silverstripe/silverstripe-elemental-fileblock/pull/43